pipeline/00-ingest.R

```diff
@@ -19,6 +19,9 @@ suppressPackageStartupMessages({
   library(noctua)
 })
 
+# Adds arrow support to speed up ingest process
+noctua_options(unload = TRUE)
+
```

Comment:
We hadn't yet switched to using the `unload = TRUE` method for `noctua`, meaning array columns were still being pulled in as comma-separated strings. Most of the updates in this file are to implement the same array handling used in the res model.